tests/instruments/test_all_instruments.py

```diff
@@ -215,6 +215,91 @@ def find_devices_in_module(module):
     "ChannelAWG",
     "ChannelAFG",
 ]
+# Instruments which do not YET define `includeSCPI` explicitly
+grandfathered_includeSCPI_instruments = [
```

Comment:
do you plan a follow-up PR that converts those instruments to the new usage? That would be good!

Reply:
I cannot do that work alone, as I do not know, which instrument is Scpi compliant.

We have to rely on users submitting that information (they will get a warning).
Later (in the restructuring), we can add a "scpiUnknownMixin" (see my other pr) for these instruments.
That will keep the code base in a working state, but will also mark these instruments as maybe not Scpi compliant.

Tdlr: no large pr planned, but in small pieces over time.

Reply:
I see (I think). So far, if not specified, we assume that SCPI inclusion is true, so assuming that the instruments in the codebase are implemented correctly, we could just carry that current behaviour over into explicit settings? Or do you want to take this opportunity to confirm for all those instruments if SCPI compatibility is _really_ appropriate?

Reply:
I don't mind shipping both PRs together.

We should raise some warning, though, that Scpi compliance is not known in order to inform users.
That will trigger people to add the information, but won't stop code from working.

Reply:
> We should raise some warning, though, that Scpi compliance is not known in order to inform users.
That will trigger people to add the information, but won't stop code from working.

The deprecation warning is that warning.